cmake/private/FairRootSummary.cmake

```diff
@@ -0,0 +1,218 @@
+################################################################################
+# Copyright (C) 2014-2021 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  #
+#                                                                              #
+#              This software is distributed under the terms of the             #
+#              GNU Lesser General Public Licence (LGPL) version 3,             #
+#                  copied verbatim in the file "LICENSE"                       #
+################################################################################
+
+function(fairroot_summary_header)
+  message(STATUS "  ")
+  message(STATUS "               C O N F I G U R A T I O N   S U M M A R Y")
```

Comment:
```suggestion
```

Too verbose IMHO.

Reply:
Commented out for now. Maybe we will find a not "too verbose" heading?